dist/angular-patternfly.js

```diff
@@ -698,6 +697,7 @@ angular.module('patternfly.charts').directive('pfDonutPctChart', ["c3ChartDefaul
           var color = '#0088CE';
 
           if (thresholds) {
+            color = '#3f9c35';
```

Comment:
That is a good idea. However, that would require a bunch of rework to
implement as it is coded to return a hex color value.  This is probably
better handled as an enhancement request.  Can you create an issue for
this? - thanks

On Thu, Sep 3, 2015 at 12:25 PM, Walden Raines <notifications@github.com>
wrote:

> In dist/angular-patternfly.js
> <https://github.com/patternfly/angular-patternfly/pull/64#discussion_r38665528>
> :
>
> > @@ -698,6 +697,7 @@ angular.module('patternfly.charts').directive('pfDonutPctChart', ["c3ChartDefaul
> >            var color = '#0088CE';
> >
> >            if (thresholds) {
> > +            color = '#3f9c35';
>
> I know that these colors were already in here but I wonder if it makes
> more sense to add a class (.threshold-error and .threshold-warning, for
> example) instead of setting these colors directly. That would allow the
> projects that consume angular-patternfly to override these colors if
> necessary.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/patternfly/angular-patternfly/pull/64/files#r38665528>
> .
>



-- 
Dave Taylor
Principle Software Engineer
(978) 392 - 1016